api/qsharp/microsoft.quantum.oracles.reflectionoraclefromdeterministicstateoracle.yml

```diff
@@ -23,6 +23,6 @@ output:
   - summary: A `ReflectionOracle` that reflects about the state $\ket{\psi}$.
     uid: microsoft.quantum.oracles.reflectionoracle
 seeAlso:
-- microsoft.quantum.canon.deterministicstateoracle
-- microsoft.quantum.canon.reflectionoracle
+- deterministicstateoracle
```

Comment:
@alan-geller, do you know why these are not getting reolved to fully-qualified UIDs?